test/OpenTelemetry.Tests/Logs/LoggerProviderBuilderBaseTests.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using Xunit;
+using static OpenTelemetry.OpenTelemetrySdk;
+
+namespace OpenTelemetry.Logs.Tests;
+
+public sealed class LoggerProviderBuilderBaseTests : IDisposable
+{
+    public LoggerProviderBuilderBaseTests()
+    {
+        Environment.SetEnvironmentVariable(SdkConfigDefinitions.SdkDisableEnvVarName, null);
+    }
+
+    public void Dispose()
+    {
+        Environment.SetEnvironmentVariable(SdkConfigDefinitions.SdkDisableEnvVarName, null);
+        GC.SuppressFinalize(this);
+    }
+
+    [Fact]
+    public void ReturnNoopLoggerProviderWhenSdkDisabledEnvVarSet()
+    {
+        Environment.SetEnvironmentVariable(SdkConfigDefinitions.SdkDisableEnvVarName, "true");
+        var builder = new LoggerProviderBuilderBase();
+
+        using var provider = builder.Build();
+
+        Assert.IsType<NoopLoggerProvider>(provider);
+    }
+
+    [Fact]
+    public void ReturnLoggerProviderSdkWhenSdkDisabledEnvVarNotSet()
+    {
+        Environment.SetEnvironmentVariable(SdkConfigDefinitions.SdkDisableEnvVarName, null);
+        var builder = new LoggerProviderBuilderBase();
+
+        using var provider = builder.Build();
+
+        Assert.IsType<LoggerProviderSdk>(provider);
+    }
```

Comment:
I think these tests could be made data-driven rather than dedicated tests per scenario:

```csharp
[Theory]
[InlineData("true", typeof(NoopLoggerProvider))]
[InlineData("false", typeof(LoggerProviderSdk))]
[InlineData(null, typeof(LoggerProviderSdk))]
public void LoggerProviderIsExpectedType(string? value, Type expected)
```